snippets/array-remove.md

```diff
@@ -1,13 +1,22 @@
 ### Array remove
 
-Use `Array.filter()` to find array elements that return truthy values and `Array.reduce()` to remove elements using `Array.splice()`.
-The `func` is invoked with three arguments (`value, index, array`).
+Finds elements that satisfy a predicate and returns them, at the same time modifying an original array.
+In other words, it works like [R#partition](http://ramdajs.com/docs/#partition), except that it mutates the original array and returns another partition instead of returning two.
```

Comment:
The description should explain *how* the snippet works, not what it does.

Reply:
But then `remove` might be a little misleading. It should be something like `withdraw`, I think.

Reply:
I absolutely agree with you that snippet names alone are not enough to explain what they do. I am already looking into added descriptions for what snippets do retroactively. For now, follow the existing guidelines and we should be able to work out the other details later down the line.